scripts/utils.js

```diff
@@ -294,25 +294,26 @@ function attachTooltip (anchor, tooltip, pos = 'right', time = 200) {
     })
 }
 
-function resetExtensionStorage () {
+function initDefaultOptions () {
   chrome.storage.sync.set({
-    agreement: false,
-    show_context: 'tab',
-    resource: false,
-    auto_update_context: false,
-    wm_count: false,
-    auto_archive: false,
-    email_outlinks: false,
     spn_outlinks: false,
     spn_screenshot: false,
-    alexa: false,
-    domaintools: false,
-    wbmsummary: false,
-    annotations: false,
-    tagcloud: false,
-    showall: false,
+    selectedFeature: null,
+    /* General */
+    wm_count: true,
+    resource: true,
+    auto_archive: true,
+    email_outlinks: true,
     not_found_popup: true,
-    selectedFeature: null
+    auto_update_context: false,
```

Comment:
Should we make it 'true', just because all other options are selected by default?

Reply:
I can push these changes if these are needed : )

Reply:
I initially tried it as true, but since it'll update Context every time a user switches tabs or opens a new page, I felt it would be too much activity going on if it were the default. I suppose I could go either way on this.

Reply:
Okay, approving the changes : )